KitePrintSDK/src/main/java/ly/kite/print/Bleed.java

```diff
@@ -0,0 +1,176 @@
+/*****************************************************
+ *
+ * Bleed.java
+ *
+ *
+ * Modified MIT License
+ *
+ * Copyright (c) 2010-2015 Kite Tech Ltd. https://www.kite.ly
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The software MAY ONLY be used with the Kite Tech Ltd platform and MAY NOT be modified
+ * to be used with any competitor platforms. This means the software MAY NOT be modified 
+ * to place orders with any competitors to Kite Tech Ltd, all orders MUST go through the
+ * Kite Tech Ltd platform servers. 
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ *****************************************************/
+
+///// Package Declaration /////
+
+package ly.kite.print;
+
+
+///// Import(s) /////
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import java.math.BigDecimal;
+import java.util.Currency;
+
+
+///// Class Declaration /////
+
+/*****************************************************
+ *
+ * This class represents a bleed.
+ *
+ *****************************************************/
+public class Bleed implements Parcelable
+  {
+  ////////// Static Constant(s) //////////
+
+  @SuppressWarnings( "unused" )
+  private static final String  LOG_TAG = "Bleed";
+
+
+  ////////// Static Variable(s) //////////
+
+  public static final Parcelable.Creator<Bleed> CREATOR =
+    new Parcelable.Creator<Bleed>()
+      {
+      public Bleed createFromParcel( Parcel sourceParcel )
+        {
+        return ( new Bleed( sourceParcel ) );
+        }
+
+      public Bleed[] newArray( int size )
+        {
+        return ( new Bleed[ size ] );
+        }
+      };
+
+
+  ////////// Member Variable(s) //////////
+
+  public int  topPixels;
```

Comment:
maybe personal preference (I always opt for defensive approach in the first instance) but these might be better as `final` so that whole class ends up immutable. Or in your experience do you prefer the following for readability?